src/main/kotlin/fi/metatavu/muisti/files/FileController.kt

```diff
@@ -19,6 +20,7 @@ import javax.inject.Singleton
  *
  * @author Antti Leppä
  */
+@Lock(time = 8, unit = TimeUnit.HOURS, value = Lock.Type.WRITE)
```

Comment:
This seems to be correct equivalent to the Java EE annotation used earlier. Although the 8 hours time seems to be waaaay too long. But that doesn't need to be resolved within this PR as this whole logic is likely to be replaced within future updates.

Reply:
I agree, 8 is too long, but it is the value from the original version, so I wonder if there was any special reason for that value :shrug: